kensu/pyspark/spark_connector.py

```diff
@@ -509,54 +489,61 @@ def init_kensu_spark(
-        dam_debug_file_level =  kwargs_or_conf_or_default('dam_debug_file_level','INFO',dam_debug_file_level, kw=kwargs, conf=kensu_conf)
-        dam_debug_file_enabled = kwargs_or_conf_or_default('dam_debug_file_enabled',False,dam_debug_file_enabled, kw=kwargs, conf=kensu_conf, tpe=bool)
-        datasource_short_name_strategy = kwargs_or_conf_or_default('datasource_short_name_strategy',None,datasource_short_name_strategy, kw=kwargs, conf=kensu_conf)
-        datasource_short_name_strategy_rules = kwargs_or_conf_or_default('datasource_short_name_strategy_rules',None,datasource_short_name_strategy_rules, kw=kwargs, conf=kensu_conf)
-        logical_datasource_name_strategy = kwargs_or_conf_or_default('logical_naming',None,logical_datasource_name_strategy, kw=kwargs, conf=kensu_conf)
-        logical_datasource_name_strategy_rules = kwargs_or_conf_or_default('logical_datasource_name_strategy_rules',None,logical_datasource_name_strategy_rules, kw=kwargs, conf=kensu_conf)
-        lineage_missing_fallback_strategy = kwargs_or_conf_or_default('lineage_missing_fallback_strategy',None,lineage_missing_fallback_strategy, kw=kwargs, conf=kensu_conf)
-        capture_spark_logs = kwargs_or_conf_or_default('capture_spark_logs',False,capture_spark_logs, kw=kwargs, conf=kensu_conf, tpe=bool)
-        h2o = kwargs_or_conf_or_default('h2o',False,h2o, kw=kwargs, conf=kensu_conf, tpe=bool)
-        add_dam_dataframe_helpers = kwargs_or_conf_or_default('add_dam_dataframe_helpers',True,add_dam_dataframe_helpers, kw=kwargs, conf=kensu_conf, tpe=bool)
-        disable_spark_writes = kwargs_or_conf_or_default('disable_spark_writes',False,disable_spark_writes, kw=kwargs, conf=kensu_conf, tpe=bool)
-        environment =  kwargs_or_conf_or_default('environment',None,environment, kw=kwargs, conf=kensu_conf)
-        fake_timestamp =  kwargs_or_conf_or_default('fake_timestamp',None,fake_timestamp, kw=kwargs, conf=kensu_conf,tpe=int)
-        projects =  kwargs_or_conf_or_default('projects',None,projects, kw=kwargs, conf=kensu_conf,tpe=list)  # type: list[str]
-        project =  kwargs_or_conf_or_default('project_names',None,project, kw=kwargs, conf=kensu_conf)
-        h2o_create_virtual_training_datasource = kwargs_or_conf_or_default('h2o_create_virtual_training_datasource',True,h2o_create_virtual_training_datasource, kw=kwargs, conf=kensu_conf, tpe=bool)
-        pandas_conversions = kwargs_or_conf_or_default('pandas_conversions',True,pandas_conversions, kw=kwargs, conf=kensu_conf, tpe=bool)
-        pandas_to_spark_df_via_tmp_file = kwargs_or_conf_or_default('pandas_to_spark_df_via_tmp_file',True,pandas_to_spark_df_via_tmp_file, kw=kwargs, conf=kensu_conf, tpe=bool)
-        pandas_to_spark_tmp_dir = kwargs_or_conf_or_default('pandas_to_spark_tmp_dir','/tmp/spark-to-pandas-tmp',pandas_to_spark_tmp_dir, kw=kwargs, conf=kensu_conf)
-
+        kensu_ingestion_url = extract_config_property('ingestion_url', None, kensu_ingestion_url, kw=kwargs, conf=kensu_conf)
```

Comment:
hmm is it `kensu_ingestion_url` in `init_kensu_spark` args, while in conf `ingestion_url` (sample config file seem to use `kensu_` prefix but extract_config_property doesn't seem to handle the kensu_ prefix)? some question for a few more properties....